lpd/qualitative_data_analysis.py

```diff
@@ -85,7 +67,7 @@ def calculate_probabilities(answers):
     """
     document = make_document(answers)
     tfidf_matrix = make_tfidf_matrix([document])
-    weights = settings.LDA_MODEL.transform(tfidf_matrix)
+    weights = settings.LDA_MODEL.transform(tfidf_matrix)[0, :]
```

Comment:
@tomaszgy Why do we need to slice here? (Just asking because I'd like to understand, not suggesting this needs to be changed.)

Reply:
@itsjeyd `transform` returns a matrix with a single row. So in order to have a list of weights (the single row mentioned before), we need to use the above Numpy's syntax.